vm-image-spec.yaml

```diff
@@ -16,7 +16,11 @@ commands:
   - name: sql-exporter
     user: nobody
     sysvInitAction: respawn
-    shell: '/bin/sql_exporter -config.file=/etc/sql_exporter.yml'
+    shell: '/bin/sql_exporter -config.file=/etc/sql_exporter.yml -web.listen-address=:9399'
+  - name: sql-exporter-autoscaling
```

Comment:
Worth considering if this should be called "autoscaling" or just "lfc" — i.e. "who it's for" vs "what it is"

Could go either way, I have no preference, just putting it out there.